src/styles/primeng-theme/components/_paginator.scss

```diff
@@ -6,7 +8,9 @@
         display: inline-flex;
         height: $paginator-page-size;
         justify-content: center;
+        margin-left: $basic-padding;
```

Comment:
This could lead to problem in other places where we use the overlay panel component, did you check that?

Reply:
yes